src/components/app/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { h, Component } from 'preact';
+import { updater, toggle, When } from '../../lib/util';
+import Fab from '../fab';
+import Header from '../header';
+// import Drawer from 'async!../drawer';
+const Drawer = require('async!../drawer').default;
+import Home from '../home';
+import * as style from './style.scss';
+
+type Props = {
+	url?: String
+}
+
+type FileObj = {
+	id: any,
+	data: any,
```

Comment:
Why `any`?

Reply:
I wasn't sure which data type was going to be easiest for the compression pipeline to consume.  For now I believe I'm reading the file into a binary String, so I'll type it as such.

- [ ] Actually type FileObj.data.